pkg/kubelet/cm/cpumanager/policy_options.go

```diff
@@ -43,6 +46,10 @@ func NewStaticPolicyOptions(policyOptions map[string]string) (StaticPolicyOption
 	for name, value := range policyOptions {
 		switch name {
 		case FullPCPUsOnlyOption:
+			if !utilfeature.DefaultFeatureGate.Enabled(kubefeatures.CPUManagerPolicyBetaOptions) {
+				return opts, fmt.Errorf("CPU Manager Policy Beta-level Options not enabled")
+			}
+
```

Comment:
Is this sufficient to gate this? It probably is for this one option, but as we add more options, it would be good to have something a bit more usable to make sure we always gate each option properly. I.e. something that lists out the options in each phase of alpha, beta, and stable all in one place, and checks the passed in options against those lists.

Something like:
```
func assertPolicyOptionAvailable(option string) error {
    alphaOptions := sets.NewString(...)
    betaOptions := sets.NewString(...)
    stableOptions := sets.NewString(...)

    if !alphaOptions.has(option) &&  !betaOptions.has(option) &&  !stableOptions.has(option) {
        return Error(...)
    }

    if alphaOptions.has(option) && !utilfeature.DefaultFeatureGate.Enabled(kubefeatures.CPUManagerPolicyAlphaOptions) {
        return Error(...)
    }

    if betaOptions.has(option) && !utilfeature.DefaultFeatureGate.Enabled(kubefeatures.CPUManagerPolicyBetaOptions) {
        return Error(...)
    }

  return nil
}
```

Reply:
I was thinking something like this. Will update the PR ASAP.